packages/eslint-plugin-mobx/package.json

```diff
@@ -45,7 +47,9 @@
         "mobx"
     ],
     "scripts": {
-        "test": "jest",
+        "test:7": "jest --config jest.config-eslint-7.js",
+        "test:9": "jest --config jest.config-eslint-9.js",
```

Comment:
Do we need seperate configs? Can't we do something like:
```
// package.json
"test:7": "jest --config jest.config.js -- 7",
"test:9": "jest --config jest.config.js -- 9"

// jest.config.js
globals: {
        ESLINT_V: process.argv[2]
}
```

Reply:
Something like that was the “Plan A”: https://github.com/mobxjs/mobx/pull/3942/commits/5c9f4bcd3d3b067fa3cb598b9b5e02be068f6625#diff-44bf0440e178d6213ede81d55d2733c534c53afb6ccdcf2451520ac2cffdf191R51

This was convenient when running "npm test" directly from the repository and required only one config (like your option).

However, CI run "npm test" from the root of the repository, and it simply runs tests for all configs of all packages, which resulted in the error: https://github.com/mobxjs/mobx/pull/3942/checks?check_run_id=31599042993 . In both my case and your the versions will be lost. Therefore, I would suggest leaving the current variant, although it requires a little duplication of code. If this is critical, we can allocate two lines to some shared config. But I think this is too much.

Reply:
I see. OK